src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/trust/TrustDataDao.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.dao.trust;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.List;
+
+public class TrustDataDao {
+
+    @Field("_id")
```

Comment:
just wondering if "_id" should just be "id". I think of "_id" as the mongo generated primary key (i might be wrong though)

Reply:
Yes, the `_` is just used for indexing, not any sort of convention for a top-level ID 👍